### ISSUE TITLE
[Question] scale_loss with multiple losses

### ISSUE THREAD

**Issue by cfzd** [NVIDIA/apex#929]
What's the difference between scaling each loss separately and scaling a weighted sum loss as a whole? Code like this:
```
with amp.scale_loss(loss0, optim) as scaled_loss:
    scaled_loss.backward()

with amp.scale_loss(loss1, optim) as scaled_loss:
    scaled_loss.backward()

with amp.scale_loss(loss2, optim) as scaled_loss:
    scaled_loss.backward()
```
and 
```
loss_all = loss0 +loss1 +loss2
with amp.scale_loss(loss_all, optim) as scaled_loss:
    scaled_loss.backward()
```

Is the second one correct? If correct, what's the difference between these two types of scaling?

**Comment by wassryan** [comment]
@cfzd hi, I met the same problem as you mentioned. I'm trying to backward 2 different losses separately as I did in https://github.com/NVIDIA/apex/issues/934#issue-675067897. Do you have any ideas?
